app/src/main/java/com/github/fribourgsdp/radio/MainActivity.kt

```diff
@@ -4,6 +4,9 @@ import android.content.Intent
 import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
 import android.widget.Button
+import android.widget.ImageButton
+
+const val USERNAME = "com.github.fribourgsdp.radio.USERNAME"
```

Comment:
You shouldn't really put that simply in the file. You should put it in a companion object of the activity you want to send it to. Then you call `ActivityName.USERNAME` here.

Reply:
Agreed, this is a temporary solution to use intents to show that the UI is dynamic. In the future, we will use preferences, but we did not have time to implement it in this sprint